src/development/tools/devtools/release-notes/release-notes-2.17.0-src.md

```diff
@@ -2,11 +2,37 @@
 
 Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
 
-## General Updates
-This release included several bug fixes and improvements, as well work
-towards some new features that are coming soon!
-See the full list of commits for DevTools 2.17.0
-[in the changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md#2170).
+## Inspector Updates
+* Added support for manually setting the package directories for your app. If you've ever 
+  loaded the Inspector and noticed that some of your widgets are not present in the 
+  Widget tree, this may be an indication that the package directories for your app 
+  have not been set or detected properly. Your package directories determine which 
+  widgets the Inspector considers to be from "your" application. If you see an 
+  empty Inspector widget tree, or if you develop widgets across multiple packages 
+  and want widgets from all these locations to show up in your tree, check the 
+  Inspector Settings dialog to ensure that your package directories are properly
+  configured. 
+  
+  [#4306](https://github.com/flutter/devtools/pull/4306)
+  
+<img
+  alt="package_directories"
+  src="{{site.url}}/development/tools/devtools/release-notes/images-2.17.0/package_directories.png"
+  width="800"
+/>
+
+## Performance Updates
+* Added frame analysis that enables hints that:
+  * Point users to the "Enhance Tracing" when a frame phase is the leading cause of slow UI time
+  * Point users to the "Raster Metrics" tab for frames with raster jank
+  * Point users to documentation when expensive operations like Canvas.saveLayer(), intrinsics calculations, and shader compilation are contributing to jank.
+  * Provide a visual breakdown of where the time in a frame was spent.
+
+  [#4339](https://github.com/flutter/devtools/pull/4339)
+  
+  Example analysis view for a janky frame:
```

Comment:
nit: I don't think we need this line. It is clear that this image is an example because it directly follows this Release Notes entry.